app/models/space_contact.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SpaceContact < ApplicationRecord
+  belongs_to :space, optional: true
+  belongs_to :space_owner
+
+  after_create_commit { broadcast_prepend_to 'space_contacts' }
+  after_update_commit { broadcast_replace_to 'space_contacts' }
+  after_destroy_commit { broadcast_remove_to 'space_contacts' }
+
+  validates :title, presence: true
```

Comment:
Would it be better to validate that it has at least one of the following?

`:telephone`, `:email`, `:url`, **or** `:description` 

The title is a nice-to-have, but some actual contact information is a must have!

Reply:
Fixed  :smile: